src/main/java/com/mongodb/kafka/connect/sink/StartedMongoSinkTask.java

```diff
@@ -42,15 +42,21 @@
 
 import com.mongodb.kafka.connect.sink.dlq.AnalyzedBatchFailedWithBulkWriteException;
 import com.mongodb.kafka.connect.sink.dlq.ErrorReporter;
+import com.mongodb.kafka.connect.util.jmx.MBeanServerUtils;
+import com.mongodb.kafka.connect.util.jmx.SinkTaskStatistics;
+import com.mongodb.kafka.connect.util.jmx.Timer;
 
 public final class StartedMongoSinkTask {
-  private static final Logger LOGGER = LoggerFactory.getLogger(MongoSinkTask.class);
```

Comment:
IIRC this was left to keep the log class the same - users may already have preferences setup specifically for the MongoSinkTask